app/styles/_core.less

```diff
@@ -503,18 +503,34 @@ label.checkbox {
 }
 
 .osc-secrets-form {
-  .picked-secret {
-    margin-bottom: 5px;
+  .remove-secret {
+    display: inline-block;
+    margin-left: 7px;
+    vertical-align: 0px;
```

Comment:
You and I are seeing very different behavior for `vertical-align` with this icon :(